frontend/.eslintcache

```diff
@@ -1 +1 @@
-[{"/app/src/index.tsx":"1","/app/src/reportWebVitals.ts":"2","/app/src/App.tsx":"3"},{"size":500,"mtime":1608362891091,"results":"4","hashOfConfig":"5"},{"size":425,"mtime":1608362891095,"results":"6","hashOfConfig":"5"},{"size":671,"mtime":1608366205319,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"1xls6ze",{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/app/src/index.tsx",[],["15","16"],"/app/src/reportWebVitals.ts",[],"/app/src/App.tsx",[],{"ruleId":"17","replacedBy":"18"},{"ruleId":"19","replacedBy":"20"},"no-native-reassign",["21"],"no-negated-in-lhs",["22"],"no-global-assign","no-unsafe-negation"]
```

Comment:
add this file to .gitignore

Reply:
arre, iske me bhi aa gyi ye file :sweat_smile:

Reply:
Done